ethtool.go

```diff
@@ -202,9 +202,9 @@ func (e *Ethtool) Close() {
 }
 
 func NewEthtool() (*Ethtool, error) {
-	fd, _, err := syscall.RawSyscall(syscall.SYS_SOCKET, syscall.AF_INET, syscall.SOCK_DGRAM, syscall.IPPROTO_IP)
-	if err != 0 {
-		return nil, syscall.Errno(err)
+	fd, err := syscall.Socket(syscall.AF_INET, syscall.SOCK_DGRAM, syscall.IPPROTO_IP)
```

Comment:
Hi, I used low level syscall in order to limit the impact of the Go bug with namespaces see https://github.com/golang/go/issues/20676

As it is fixed in 1.10 I think that OK now.